tests/framework/results/test_context.py

```diff
@@ -385,22 +385,6 @@ def test_gather_results_with_no_stratifications():
     assert len(list(ctx.gather_results(population, event_name))) == 1
 
 
-def test__format_results():
```

Comment:
No more `ResultsContext._format_results()` method. The primary purpose of that was to convert the stratified/aggregated dataframe into the long key:value pairs that eventually becamse output.hdf columns.